### PR TITLE
generate the static graph code of some ops

### PR DESCRIPTION
**Pull request by heavyrain-lzy**
### PR types
Others

### PR changes
Ops

### Describe
Generate the static graph of frame, gather_nd, isclose, overlap_add, pixel_shuffle base yaml config.

### CODE REVIEW

**Review by YuanRisheng — DISMISSED** [review]

**Review by jiahy0825 — DISMISSED** [review]

File: paddle/fluid/operators/gather_nd_op.cc [hunk 1]
```diff
@@ -1,150 +0,0 @@
...
- protected:
-  framework::OpKernelType GetExpectedKernelType(
-      const framework::ExecutionContext& ctx) const override {
-    auto* x = ctx.Input<phi::DenseTensor>("X");
-    const auto& x_type = OperatorWithKernel::IndicateVarDataType(ctx, "X");
-    return framework::OpKernelType(
-        x_type,
-        x_type == framework::proto::VarType::BOOL
-            ? x->place()  // to be consistent with compare and logical ops
-            : ctx.device_context().GetPlace());
```
Comment (zyfncg) [inline paddle/fluid/operators/gather_nd_op.cc:36]: 这里可能需要留意下是否有特殊case会受影响

File: paddle/fluid/operators/pixel_shuffle_op.cc [hunk 2]
```diff
@@ -1,115 +0,0 @@
...
-REGISTER_OPERATOR(pixel_shuffle_grad,
-                  ops::PixelShuffleGradOp,
-                  PixelShuffleGradInferShapeFunctor);
-
-REGISTER_OP_VERSION(pixel_shuffle)
-    .AddCheckpoint(
-        R"ROC(
-               Compatible upgrade of pixel_shuffle, add a new attribute [data_format])ROC",
-        paddle::framework::compatible::OpVersionDesc().NewAttr(
-            "data_format", "Specify the data format of the input data", true));
```
Comment (zyfncg) [inline paddle/fluid/operators/pixel_shuffle_op.cc:115]: 这个需要在`op_version.yaml`加下配置

Comment (heavyrain-lzy) [reply paddle/fluid/operators/pixel_shuffle_op.cc:115]: Done.Thx.

File: paddle/fluid/operators/gather_nd_op.cc (hunk 1 above)
Comment (heavyrain-lzy) [reply paddle/fluid/operators/gather_nd_op.cc:36]: 已标记，后续会持续关注

**Review by chenwhql — DISMISSED** [review]

**Review by zyfncg — DISMISSED** [review]

File: python/paddle/tensor/logic.py [hunk 3]
```diff
@@ -957,20 +957,28 @@ def bitwise_not(x, out=None, name=None):
 
 @templatedoc()
 def isclose(x, y, rtol=1e-05, atol=1e-08, equal_nan=False, name=None):
-    """
-    ${comment}
+    r"""
+    This operator checks if all :math:`x` and :math:`y` satisfy the condition:
```
Comment (Ligoml) [inline python/paddle/tensor/logic.py]:
```suggestion
    Check if all :math:`x` and :math:`y` satisfy the condition:
```

Comment (Ligoml) [reply python/paddle/tensor/logic.py]: 尽量在api文档中避免出现op的错误描述

Comment (heavyrain-lzy) [reply python/paddle/tensor/logic.py]: Done.Thx!

**Review by chenwhql — APPROVED** [review]

**Review by Ligoml — APPROVED** [review]
LGTM for docs

**Review by zyfncg — APPROVED** [review]

**Pull request merged**